src/zino/statemodels.py

```diff
@@ -135,6 +135,13 @@ class LogEntry(BaseModel):
     timestamp: datetime.datetime = Field(default_factory=now)
     message: str
 
+    def model_dump_legacy(self) -> List[str]:
+        """Returns the contents of this log entry in the text format expected on the Zino legacy server protocol"""
```

Comment:
Instead of "in the text format expected on the Zino legacy server protocol" could you show an example?

Reply:
.. otherwise, show examples in the tests so that it will be less necessary to run pdb to see what is passed around later on.

Reply:
The tests are already meant to document the expected output, but sure I can update the docstring; see update.